web/react/components/view_image.jsx

```diff
@@ -213,7 +224,7 @@ module.exports = React.createClass({
             content = (
                 <div>
                     <img className='loader-image' src='/static/images/load.gif' />
-                    { percentage > 0 ?
+                    {percentage > 0 ?
```

Comment:
Do you mind getting rid of the last ternary statement in here? It looks like the rest of the file matches the style guide except for that.